### PR TITLE
General markdown file cleanup

### PR DESCRIPTION
**Pull request by kevmoo**
Used auto-fix features in VS Code
Standardized and fixed shell code blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,21 +8,21 @@
 > [issues](https://github.com/GoogleCloudPlatform/functions-framework-dart/issues)
 > for bugs.
 
-|Functions Framework|Unit Tests|Lint Test|Conformance Tests|
-|---|---|---|---|
-[Dart][ff_dart]| [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
+| Functions Framework | Unit Tests                                 | Lint Test                                  | Conformance Tests                                        |
+| ------------------- | ------------------------------------------ | ------------------------------------------ | -------------------------------------------------------- |
```
Comment (grant) [inline README.md:12]: I'd revert this. 3 en dashes is common and more maintainable.

File: README.md [hunk 2]
```diff
@@ -36,25 +36,25 @@ Response function(Request request) => Response.ok('Hello, World!');
 
 To:
 
-```sh
-curl http://my-url
+```shell
+$ curl https://example.com
```
Comment (grant) [inline README.md:40]: Don't include $ in bash scripts. They aren't copy pastable.

**Review by grant — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kevmoo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (kevmoo) [reply README.md]: Removed this one. Should chat w/ @subfuzion if we want to clean up the rest in the repo

**Comment by kevmoo** [comment]
@grant – need another nod here

**Review by grant — APPROVED** [review]

**Pull request merged**